web/src/lib/domain-logic/routes.ts

```diff
@@ -53,7 +53,6 @@ export enum QUERIES {
   industry = "industry",
   sector = "sector",
   code = "code",
-  openTaxFilingsModal = "openTaxFilingsModal",
   flow = "flow",
```

Comment:
pebble - I think we cna safely remove reference to openTaxFilingsModal on line 34